### PR TITLE
added general .travis.yaml file

### LINKED ISSUE #24: Enable Travis CI for our repository
Before opening a feature request against this repo, consider whether the feature should/could be implemented in the [other OpenTelemetry client libraries](https://github.com/open-telemetry). If so, please [open an issue in opentelemetry-specification](https://github.com/open-telemetry/opentelemetry-specification/issues/new) first.

**Is your feature request related to a problem?**
It is not.  We'd like to have automated test suites for commits

**Describe the solution you'd like**
We'd like to have CI run on PR open

**Describe alternatives you've considered**
Manual testing

### PR DESCRIPTION
**Pull request by VictorAvelar**
## Describe your changes

Created the `.travis.yaml` file which configures the pipeline for CI using travis.org, the file adds support for PHP 7.2 and 7.3 and also uses the scrutinizer `ocular.phar` to upload the tests coverage result generated when running the tests suite.

Fixes #24

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+dist: trusty
+language: php
+
+php:
+    - 7.2
+    - 7.3
+
```
Comment (bobstrecansky) [inline .travis.yml]: We need to remember to add php 7.4 here in 2 weeks!

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+dist: trusty
```
Comment (bobstrecansky) [inline .travis.yml]: Did you choose 'trusty' on purpose here?  This should be fine, we've just been using a debian image in our test suite.

Comment (VictorAvelar) [reply .travis.yml]: https://docs.travis-ci.com/user/reference/overview/#virtualization-environments

I try to upgrade it to xenial or bionic, but this one works well, just let me know and I can give it a shot.

File: .travis.yml (hunk 1 above)
Comment (VictorAvelar) [reply .travis.yml]: Sure, I will create a reminder now 😄

**Comment by bobstrecansky** [comment]
We can also probably eventually add the phan tests that I'm adding here: https://github.com/open-telemetry/opentelemetry-php/pull/27

File: .travis.yml (hunk 2 above)
Comment (bobstrecansky) [reply .travis.yml]: I think this is good for now.  We can change it later if we see it causes strife.

**Pull request merged**